mopidy_musicbox_webclient/static/js/controls.js

```diff
@@ -236,7 +236,7 @@ function removeTrack() {
     }
     var track = {};
     track.uri = [currentplaylist[i].uri];
-    mopidy.tracklist.remove(track);
+    mopidy.tracklist.remove({'uri':track.uri});
```

Comment:
It's interesting that the old code worked for you. Based on the [Mopidy API docs] ( https://docs.mopidy.com/en/latest/api/core/#mopidy.core.TracklistController.filter) I would expect 

```mopidy.tracklist.remove(uri = track.uri)``` 

to work as well (which is probably easier to read), but not just passing ```track```.